RacecarSim/Assets/Scripts/NonMonoBehavior/PythonInterface.cs

```diff
@@ -40,7 +40,7 @@ public class PythonInterface
     /// <summary>
```

Comment:
Above, you need to change `private const int version = 1;` to 2 because you are updating the communication protocol